VP-fitness-frontend/app/build.gradle

```diff
@@ -24,7 +24,18 @@ dependencies {
     androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-    compile 'com.android.support:appcompat-v7:25.3.1'
+    compile project(":fontutil")
+    compile 'com.android.support:appcompat-v7:26.0.0-alpha1'
     compile 'com.android.support.constraint:constraint-layout:1.0.2'
+    compile 'com.android.support:design:26.0.0-alpha1'
+    compile 'com.rengwuxian.materialedittext:library:2.1.4'
+    compile 'com.android.support:support-v4:26.0.0-alpha1'
+    compile 'com.matthew-tamlin:sliding-intro-screen:3.2.0'
```

Comment:
Note that all of these reviews usually reference the line above this review comment (i.e. in this case line 33, not below on line 34)